lib/project_types/script/messages/messages.rb

```diff
@@ -5,88 +5,86 @@ module Messages
-          eacces_help: "Try again and choose a different directory.",
+          eacces_help: "Get permission for this directory or choose a different one.",
 
           enospc_cause: "You don't have enough disk space to do this action.",
-          enospc_help: "Free up some space and try again.",
+          enospc_help: "Get more space.",
 
           oauth_cause: "Something went wrong while authenticating your account with the Partner Dashboard.",
-          oauth_help: "Try again.",
+          oauth_help: "Wait a few minutes and try again.",
 
-          invalid_context_cause: "Your .shopify-cli.yml file is not correct. Values are missing for "\
+          invalid_context_cause: "Your .shopify-cli.yml is formatted incorrectly. It’s missing values for "\
                                  "extension_point_type or script_name.",
           invalid_context_help: "Add these values and try again.",
 
-          invalid_script_name_cause: "Invalid script name.",
-          invalid_script_name_help: "Replace or remove unsupported characters. Valid characters "\
-                                    "are numbers, letters, hyphens, or underscores.",
+          invalid_script_name_cause: "Script name contains unsupported characters.",
+          invalid_script_name_help: "Use only numbers, letters, hyphens, or underscores.",
 
-          no_existing_apps_cause: "You don't have any apps in your Partner Dashboard.",
-          no_existing_apps_help: "Create an app with {{command:shopify [node|rails] create}}" \
-                                 " or visit https://partners.shopify.com/.",
+          no_existing_apps_cause: "Your script can’t be pushed to an app because your Partner account doesn’t have any apps.",
+          no_existing_apps_help: "Create an app.",
 
-          no_existing_orgs_cause: "You don't have any partner organizations.",
-          no_existing_orgs_help: "Visit https://partners.shopify.com/ to create a partners account.",
+          no_existing_orgs_cause: "Your account doesn’t belong to a Partner Organization",
+          no_existing_orgs_help: "Visit https://partners.shopify.com/ to create an account.",
 
           project_exists_cause: "A directory with this same name already exists.",
-          project_exists_help: "Try again and enter a different name for the script.",
+          project_exists_help: "Choose a different name for your script.",
 
-          invalid_extension_cause: "Invalid Script API %s.",
-          invalid_extension_help: "Allowed values: %s.",
+          invalid_extension_cause: "The name of the Script API is incorrect: %s.",
+          invalid_extension_help: "Supported values: %s.",
 
-          invalid_language_cause: "Invalid language %s.",
-          invalid_language_help: "Allowed values: %s.",
+          invalid_language_cause: "The language is not supported: %s.",
+          invalid_language_help: "Choose a supported language: %s.",
```

Comment:
Nit that you can ignore if you want, but any reason we say `Choose a supported language:` here but `Supported values:` on line 43, when the script API is incorrect?